### PR TITLE
PLAT-99533: Refine VirtualList and Scroller JS Docs

### PR DESCRIPTION
**Pull request by MikyungKim**
Updated JS Docs for VirtualList and Scroller

### CODE REVIEW

**Comment by ybsung** [comment]
Empty line before `* @example`, please.

https://github.com/enactjs/agate/pull/218/files#diff-e7aae00fbc8467587b10ae4ec74df176R2

File: Scroller/Scroller.js [hunk 1]
```diff
@@ -79,8 +80,87 @@ let Scroller = (props) => {
...
+	/**
+	 * This is set to `true` by SpotlightContainerDecorator
+	 *
+	 * @type {Boolean}
+	 * @private
+	 */
+	'data-spotlight-container': PropTypes.bool,
+
+	/**
+	 * `false` if the content of the list or the scroller could get focus
```
Comment (ybsung) [inline Scroller/Scroller.js]: Would you omit the `the list or ` here?

File: Scroller/Scroller.js [hunk 2]
```diff
@@ -112,19 +377,24 @@ Scroller.propTypes = /** @lends agate/Scroller.Scroller.prototype */ {
 };
 
 Scroller.defaultProps = {
-	'data-spotlight-container-disabled': false, // eslint-disable-line react/default-props-match-prop-types
+	'data-spotlight-container-disabled': false,
+	cbScrollTo: nop,
 	direction: 'both',
-	focusableScrollbar: false, // eslint-disable-line react/default-props-match-prop-types
+	focusableScrollbar: false,
 	horizontalScrollbar: 'auto',
```
Comment (ybsung) [inline Scroller/Scroller.js]: Do we need a `noScrollByDrag: false` here?

File: VirtualList/VirtualList.js [hunk 3]
```diff
@@ -173,25 +455,50 @@ VirtualList.propTypes = /** @lends agate/VirtualList.VirtualList.prototype */ {
 	 * @default 'auto'
 	 * @public
 	 */
-	verticalScrollbar: PropTypes.oneOf(['auto', 'visible', 'hidden'])
+	verticalScrollbar: PropTypes.oneOf(['auto', 'visible', 'hidden']),
+
+
```
Comment (ybsung) [inline VirtualList/VirtualList.js]: Double empty lines.

File: Scroller/Scroller.js (hunk 2 above)
Comment (MikyungKim) [reply Scroller/Scroller.js]: Actually we are setting `noScrollByDrag: noScrollByDrag: !platform.touchscreen,` in `useScroll` so I'm going to remove this prop here.

File: VirtualList/VirtualList.js [hunk 4]
```diff
@@ -327,11 +826,95 @@ VirtualGridList.propTypes = /** @lends agate/VirtualList.VirtualGridList.prototy
...
+	/**
+	 * Specifies how to scroll.
+	 *
+	 * Valid values are:
+	 * * `'translate'`,
+	 * * `'native'`.
+	 *
+	 * @type {String}
+	 * @default 'native'
```
Comment (ybsung) [inline VirtualList/VirtualList.js]: The default value for the agate/VirtualList scrollMode is 'translate'.

**Pull request merged**